hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/PlanExecutor.java

```diff
@@ -479,12 +480,17 @@ private List<Object> prepareArguments(QueryParameterMetadata parameterMetadata,
     }
 
     private static int findQueryExceptionCode(Throwable t) {
+        Throwable orig = t;
         while (t != null) {
             if (t instanceof QueryException) {
                 return ((QueryException) t).getCode();
             }
+            if (ExceptionUtil.isTopologyException(t)) {
```

Comment:
nit: personally I'd prefer to use static import here, just looks better if (isTopologyException(t)). isTopologyException is not a often-repeated name and ExceptionUtil prefix doesn't add much info when reading

Reply:
done